docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Mint NFTs on Avalanche 🚧
+description: Learn how to start minting NFTs on Avalanche.
+issueUrl: https://github.com/protocol/nft-website/issues/224
+---
+
+# Mint NFTs on Avalanche
+
+This short tutorial will quickly guide you through getting started with an EVM-compatible NFT minting work flow on [Avalanche](https://www.avax.network/)using Node.js REPL. This is quite useful if you are migrating from Ethereum or other EVM-compatible blockchain and wish to reuse your code.
+
+## Avalanche vs Ethereum
+
+Unlike Ethereum, Avalanche is a network of multi-blockchains, one of which runs a fork of Ethereum Virtual Machine (EVM) and compatible with Ethereum.
+
+Avalanche consists of 3 subnets, X-chain, P-chain, and C-chain.
+
+- **X-chain**: Deals with exchanges of value and runs AVM (namespaced `avm`)
+- **P-chain**: Deals with platform/protocol (core) and able to create new arbitrary blockchains (namespaced `platform`)
+- **C-chain**: the EVM-compatible chain capable of running Solidity smart contract / dapps. It has Ethereum-compatible addresses (hexadecimal strings prefixed with "0x" concatenated with the rightmost 20 bytes of the Keccak-256 hash ECDSA public key)
+
+Most beginner's confusion will be from distinguishing these different subnets. Note that only the C-chain is EVM-compatible and has Ethereum-compatible addresses. and most of dapps will be interacting with this chain.
+
+![Avalanche's diagram of subnets](https://docs.avax.network/assets/images/image(21)-3c5cb7f1f21926b05ae3631f453ed49d.png)
+
+## Setting up
+
+The quickest way to start is to run a group of simulator nodes locally. To do that, follow these steps:
```

Comment:
I am a little confused about local simulator nodes? Mainly:

1. Does local nodes also consist of subnets?
2. If yes, how do I specify which subnet I am interacting with?
3. when I create a wallet with the pre-funded token, which subnets it is? How do I transfer tokens to C-subnet if I need to play with c-subnet in the local network?

I will suggest to add some more intro and detailed info for local simulation netowrk.